src/Apps/Collect/CollectApp.tsx

```diff
@@ -36,6 +41,8 @@ export class CollectApp extends Component<CollectAppProps> {
         <Meta property="og:description" content={description} />
         <Meta property="twitter:description" content={description} />
         <Link rel="canonical" href={canonicalHref} />
+        <BreadCrumbList items={[{ path: "/collect", name: "Collect" }]} />
```

Comment:
I'm wondering if we should expand this to some how handle the case for when there is a medium for the collect page? I think so because there is mention of it in the ticket under "For Collect/Medium, the breadcrumb structure can be:". 

This would handle url such as `/collect/painting`. I think you could pass in `canonicalHref` which takes this into account.

Reply:
Thanks for catching this! Yeah there should probably another path for mediums. I'll update this component so all mediums will be crawled.

Reply:
Now this is tested with `CollectApp.test.tsx` and covers tow cases:

 * `Artsy > Collect` on https://www.artsy.net/collect
 * `Artsy > Collect > Painting` on https://www.artsy.net/collect/painting